scripts/index.js

```diff
@@ -1,13 +1,9 @@
 // add your solutions here
 function isPositive(nums){
-    result = []
-    nums.forEach(function (num){
-        if (num > 0){
-            result.push(num)
-        }
-    })
-    return result
+    results = []
+    const isPositive = (num) => num > 0;
```

Comment:
Nice. Giving the arrow function a variable name makes the intention even more clear.